test/internal/packagemanifests.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Modified from https://github.com/kubernetes-sigs/kubebuilder/tree/39224f0/test/e2e/v3
+
+package internal
+
+import (
+	"io/ioutil"
+	"path/filepath"
+)
+
+// Makefile fragments to add to the base Makefile just to ensure the packagemanifests feature
+const makefilePackagemanifestsNonGoFragment = `
+# Options for "packagemanifests".
+ifneq ($(origin CHANNEL), undefined)
+PKG_CHANNELS := --channel=$(CHANNEL)
+endif
+ifeq ($(IS_CHANNEL_DEFAULT), 1)
+PKG_IS_DEFAULT_CHANNEL := --default-channel
+endif
+PKG_MAN_OPTS ?= $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)
+
+# Generate package manifests.
+packagemanifests: kustomize
+	operator-sdk generate kustomize manifests -q --interactive=false
```

Comment:
Why do we specify `--interactive=false` here, but not for Go?

Reply:
We have a target for GO and for NonGO projects. It does NOT equal for both. See: https://v1-0-x.sdk.operatorframework.io/docs/olm-integration/generation/#package-manifests-formats

Reply:
Because, the packagemanifests will call the bundle if it was not generated yet. 

```
$ make packagemanifests 
operator-sdk generate kustomize manifests -q

Display name for the operator (required): 
> 
```